xarray/test/test_variable.py

```diff
@@ -213,7 +213,10 @@ def test_pandas_period_index(self):
     def test_1d_math(self):
         x = 1.0 * np.arange(5)
         y = np.ones(5)
-        v = self.cls(['x'], x)
+
+        # should we need `.to_base_variable()`?
+        # probably a break that `+v` changes type?
+        v = self.cls(['x'], x).to_base_variable()
```

Comment:
I think this it's OK that any arithmetic converts `IndexVariable` to `Variable`. That seems pretty consistent to me.